app/scripts/templates/settings/devices.mustache

```diff
@@ -20,11 +20,14 @@
         {{#devices}}
           <li class="device {{type}}" id="{{id}}">
             <div class="device-name">
-              {{name}} {{#isCurrentDevice}}<span class="device-current">{{#t}}(current){{/t}}</span>{{/isCurrentDevice}}
+              {{name}}
             </div>
+            {{#isCurrentDevice}}<span class="device-current">{{#t}}current device{{/t}}</span>{{/isCurrentDevice}}
```

Comment:
I find having it all on one line more difficult to read than before, esp since lines 26-30 are on distinct lines. The original HTML was all on one line to because spacing was wack otherwise, with the new HTML there is no need for everything to be smushed together like this.